addons/account/models/partner.py

```diff
@@ -460,9 +461,13 @@ def _get_company_currency(self):
             else:
                 partner.currency_id = self.env.company.currency_id
 
-    credit = fields.Monetary(compute='_credit_debit_get', search=_credit_search,
+    credit = fields.Monetary(compute='_compute_debit_credit', search=_credit_search,
         string='Total Receivable', help="Total amount this customer owes you.",
         groups='account.group_account_invoice,account.group_account_readonly')
+    credit_to_invoice = fields.Monetary(
+        compute='_compute_debit_credit',
+        groups='account.group_account_invoice,account.group_account_readonly'
+    )
```

Comment:
I thought a computed field was a bit cleaner, but a method would do as well. I created a method instead now.

Reply:
Both are OK, but I think you are right, it will be better to use a computed field.
Once computed, the result will be cached, so it won't be recomputed every time the invoice total changes.
Sorry about that, I should have thought about it earlier :sweat_smile:

Reply:
No problem. I changed it again to a computed field.